nextcloud-startup-script.sh

```diff
@@ -374,41 +374,33 @@ sleep 3
 clear
 
 # Extra configurations
-whiptail --title "Extra configurations" --checklist --separate-output "Choose what you want to configure\nSelect by pressing the spacebar" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+choice=$(whiptail --title "Extra configurations" --checklist "Choose what you want to configure\nSelect by pressing the spacebar" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 "Security" "(Add extra security based on this http://goo.gl/gEJHi7)" OFF \
 "Static IP" "(Set static IP in Ubuntu with netplan.io)" OFF \
 "Automatic updates" "(Automatically update your server every week on Sundays)" OFF \
-"CookieLifetime" "(Configure forced logout timeout for users using the web GUI)" OFF 2>results
+"CookieLifetime" "(Configure forced logout timeout for users using the web GUI)" OFF 3>&1 1>&2 2>&3)
 
-while read -r -u 4 choice
-do
-    case "$choice" in
-        "Security")
-            clear
-            run_static_script security
-        ;;
-
-        "Static IP")
-            clear
-            run_static_script static_ip
-            rm -f "$SCRIPTS"/lib.sh
-        ;;
-
-	"Automatic updates")
-            clear
-            run_static_script automatic_updates
-        ;;
-
-        "CookieLifetime")
-            clear
-            run_static_script cookielifetime
-        ;;
-
-        *)
-        ;;
-    esac
-done 4< results
-rm -f results
+case *"$choice" in
```

Comment:
@enoch85 I think here is a typo...

Reply:
Did you test, and it didn't work? If so, please comitt a PR.

Thanks! :+1:

Reply:
I have tested it and it works. So no need to fix that.

Reply:
doesn't it look the same everywhere else? It should be consistent IMO.

Reply:
no, this is different (was a mistake) I could open a PR to fix that, but this is a bit to less for a PR imo. I will put it in https://github.com/nextcloud/vm/pull/1072

Reply:
this is fixed here: https://github.com/nextcloud/vm/pull/1072/commits/4535ec6ef8376be9101e6abad68eb77f4681db41